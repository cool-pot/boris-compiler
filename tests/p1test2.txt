*** scanner test 2
*** this comment contains something that looks like legal code: a=1; b=a+b;
*** but it's still just a comment!!!!

*** this is just garbage...
####!@#%^}}{
\
\\!#@!$
*** end of garbage