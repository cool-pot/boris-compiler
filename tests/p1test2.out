[Token 297] in Line 1, [1,18].
[Token 299] in Line 2, [0,0].
[Token 297] in Line 2, [1,75].
[Token 299] in Line 3, [0,0].
[Token 297] in Line 3, [1,37].
[Token 299] in Line 4, [0,0].
[Token 299] in Line 5, [0,0].
[Token 297] in Line 5, [1,27].
[Token 299] in Line 6, [0,0].
[Token 303] in Line 6, [1,1].
[Token 303] in Line 6, [2,2].
[Token 303] in Line 6, [3,3].
[Token 303] in Line 6, [4,4].
[Token 303] in Line 6, [5,5].
[Token 303] in Line 6, [6,6].
[Token 303] in Line 6, [7,7].
[Token 303] in Line 6, [8,8].
[Token 303] in Line 6, [9,9].
[Token 303] in Line 6, [10,10].
[Token 303] in Line 6, [11,11].
[Token 303] in Line 6, [12,12].
[Token 299] in Line 7, [0,0].
[Token 303] in Line 7, [1,1].
[Token 299] in Line 8, [0,0].
[Token 303] in Line 8, [1,1].
[Token 303] in Line 8, [2,2].
[Token 303] in Line 8, [3,3].
[Token 303] in Line 8, [4,4].
[Token 303] in Line 8, [5,5].
[Token 303] in Line 8, [6,6].
[Token 303] in Line 8, [7,7].
[Token 299] in Line 9, [0,0].
[Token 297] in Line 9, [1,18].
